packages/shared/src/components/squads/moderation/SquadEmptyScreen.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, ReactElement } from 'react';
+import { IconProps, IconSize } from '../../Icon';
+import {
+  TypographyColor,
+  Typography,
+  TypographyType,
+} from '../../typography/Typography';
+
+interface SquadEmptyScreenProps {
+  Icon: FC<IconProps>;
+  title: string;
+  description: string;
+}
+
+export function SquadEmptyScreen({
```

Comment:
Would suggest to remove Squad from naming, not necessary, it's already within squads directory, and same comment for the rest

Reply:
Yeah, but normally, the main function's name should be the same as with the file and we do named export, removing the Squad on the name would cause some naming issues down the line.